R/pkg/inst/tests/testthat/test_sparkSQL.R

```diff
@@ -2083,6 +2083,42 @@ test_that("dapply() on a DataFrame", {
   expect_identical(expected, result)
 })
 
+test_that("Window functions on a DataFrame", {
+  ssc <- callJMethod(sc, "sc")
+  hiveCtx <- tryCatch({
+    newJObject("org.apache.spark.sql.hive.test.TestHiveContext", ssc)
+  },
+  error = function(err) {
+    skip("Hive is not build with SparkSQL, skipped")
+  })
+
```

Comment:
I think we need another JIRA for changing hiveContext to sparkSession in SparkR API?